doc/nrf/ug_thingy91.rst

```diff
@@ -163,12 +163,12 @@ You can choose the method based on the availability or absence of an external de
 
 .. _build_pgm_vsc:
 
-Building and programming using VS Code extension
-================================================
+Building and programming using |nRFVSC|
```

Comment:
```suggestion
Building and programming using |VSC|
```
We should keep this consistent with the other guides and use |VSC|. It makes more sense to talk about the IDE as a whole because the following section is focused around the command line. It puts more emphasis on the tool preference.

Reply:
Should it be changes in the user guides as well? E.g., 
![image](https://user-images.githubusercontent.com/51916561/171421090-23eb56b3-0d56-47e5-87b6-f7831f03d4bc.png)

Or only for headings?

Reply:
I'd keep it as nRFVSC in the line 463, because we provide the extra argument in the extension. Correct, @Avalei ?

Reply:
It is fine to keep as |nRFVSC| in that example because we are talking about the extension specifically and referencing sections in the extension interface itself.